router_macro_derive/src/lib.rs

```diff
@@ -248,7 +248,7 @@ pub fn define_as_root(item: TokenStream) -> TokenStream {
 ///
 ///
 /// #[derive(Debug, PartialEq, Clone, RoutingModules)]
-/// #[modules_path = "pages"]
+/// #[modules::path = "pages"]
```

Comment:
oops. This slipped through. can delete this line-change

Reply:
I let you doing it :wink: